angelsrefining/settings.lua

```diff
@@ -127,5 +127,12 @@ data:extend(
     maximum_value = 1000,
     order = "d"
   },
+  {
+    type = "bool-setting",
+    name = "angels-refining-new-sorting",
+    setting_type = "startup",
+    default_value = false,
```

Comment:
Should default to `true`

Reply:
To keep the discussion on one place, putting this here as reference
![image](https://user-images.githubusercontent.com/26593477/121220330-cf0e6f00-c884-11eb-9c47-f0193895f2ef.png)

Reply:
i have a draft to remove the setting already, i was going to wait to look at the catalyst sorting options before i push it though...
Do we want a full re-work, or just shuffle the 3 offending materials around?

Reply:
Could you maybe push it already? Meanwhile, you could start working on the smelting prerequisites as that only depends on pure sorting. Meanwhile we can discuss catalyst sorting in #571